docs/types/mapping-types.rst

```diff
@@ -52,6 +58,34 @@ each ``_KeyType``, recursively. For example with a mapping:
         }
     }
 
+The example below uses a mapping type inside another mapping type. The important
+point to remember are that the mapping inside the mapping is not a return variable,
+but the variable inside it is.
+
+::
+
+    pragma solidity >=0.4.0 <0.7.0;
+
+    contract MappingExample {
+        mapping(address => mapping(address => bool)) public validBalances;
+
+        function transfer(address sender, address recipient) view public {
+            require(validBalances[sender][recipient]);
+        }
+
+        function allowTransfers(address recipient) public {
+            validBalances[msg.sender][recipient] = true;
+        }
+
+    }
+
+    contract MappingUser {
+        function f() public returns (bool) {
+            MappingExample m = new MappingExample();
+            m.transfer(address(this),address(this));
```

Comment:
```suggestion
            m.transfer(address(this), address(this));
```

Reply:
maybe at least transfer from this to msg.sender? Or provide an address as a parameter of `f()`? Then you do not need the type conversions.